client/flutter/lib/generated/intl/messages_en.dart

```diff
@@ -49,13 +70,21 @@ class MessageLookup extends MessageLookupByLibrary {
     "homePagePageSliverListSupport" : MessageLookupByLibrary.simpleMessage("Help support the relief effort"),
     "homePagePageSubTitle" : MessageLookupByLibrary.simpleMessage("Virus response & tools"),
     "homePagePageTitle" : MessageLookupByLibrary.simpleMessage("Coronavirus"),
+    "legalLandingPageAnd" : MessageLookupByLibrary.simpleMessage("and"),
+    "legalLandingPageButtonAgree" : MessageLookupByLibrary.simpleMessage("By proceeding, you agree to our"),
+    "legalLandingPageButtonGetStarted" : MessageLookupByLibrary.simpleMessage("Get Started"),
+    "legalLandingPagePrivacyPolicyLinkText" : MessageLookupByLibrary.simpleMessage("Privacy Policy"),
+    "legalLandingPagePrivacyPolicyLinkUrl" : MessageLookupByLibrary.simpleMessage("https://whocoronavirus.org/privacy"),
+    "legalLandingPageTermsOfServiceLinkUrl" : MessageLookupByLibrary.simpleMessage("https://whocoronavirus.org/terms"),
+    "legalLandingPageTitle" : MessageLookupByLibrary.simpleMessage("Official WHO COVID-19 Information App"),
+    "locationSharingPageBackgroundImage" : MessageLookupByLibrary.simpleMessage("assets/onboarding/onboarding_location.png"),
     "locationSharingPageButton" : MessageLookupByLibrary.simpleMessage("Allow Location Sharing"),
     "locationSharingPageDescription" : MessageLookupByLibrary.simpleMessage("To get local news and information, allow location sharing."),
     "locationSharingPageTitle" : MessageLookupByLibrary.simpleMessage("Get the latest news from your community"),
     "newsFeedSliverListNewsFeedItemDescription1" : MessageLookupByLibrary.simpleMessage("Situation reports provide the latest updates on the novel coronavirus outbreak."),
     "newsFeedSliverListNewsFeedItemDescription2" : MessageLookupByLibrary.simpleMessage("Rolling updates on coronavirus disease (COVID-19) sourced from across WHO media."),
     "newsFeedSliverListNewsFeedItemDescription3" : MessageLookupByLibrary.simpleMessage("All news releases, statements, and notes for the media."),
-    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-19) press briefings including videos, audio and transcripts."),
+    "newsFeedSliverListNewsFeedItemDescription4" : MessageLookupByLibrary.simpleMessage("Coronavirus disease (COVID-2019) press briefings including videos, audio and transcripts."),
```

Comment:
COVID-19 is the official name we must use

Reply:
done, thanks for spotting that